docs/get_started/raspbian_setup.md

```diff
@@ -36,38 +36,37 @@ Clone the MXNet source code repository to your computer, using ```git```.
     git clone https://github.com/dmlc/mxnet.git --recursive
 ```
 
-Then build the full MxNet library, ensuring the ARM flag is enabled (this removes the x86 specific msse flag in the NNVM Submodule's Makefile, making it ARM compatible)
+If you are not processing images with MxNet on the Pi you can build MxNet without opencv to minimize the size of the compiled library. You can do this by compiling with the following code:
```

Comment:
A little pedantic but 'code' should actually be 'command' in the context of bash